### PR TITLE
Added loading icon

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #1045

### CODE REVIEW

**Comment by mmmavis** [comment]
@xmatthewx will this icon work? If not I can change it to what we have on Pulse.

https://foundation-mofostaging-pr-1093.herokuapp.com/fellowships/directory/

loading-2

/cc @taisdesouzalessa

**Comment by xmatthewx** [comment]
Looks good!

**Comment by mmmavis** [comment]
@gvn I'm too lazy to go on Heorku to whitelist this review app. Let me know if seeing the loading icon in action is enough for you to do the code review. This [gif](https://github.com/mozilla/foundation.mozilla.org/pull/1093#issuecomment-368178089) might help a bit too.

**Comment by gvn** [comment]
It feels like this could be done in a much simpler way with a single SVG asset rotated with CSS. Is there some reason it's built like this?

**Comment by mmmavis** [comment]
@gvn [slide 5](https://docs.google.com/presentation/d/13dl3c8Ar2lHf15gmI6lXtSNV0i9L8JI6qUQpI6M1ETw/edit#slide=id.g3212bd4c5b_0_33) is what we were hoping to achieve but we didn't have the time to get the "scaling dots" to work... that's why we are going with a simpler version (for now). 

I'm fine with either sticking with this pure CSS version or SVG (3 dots on circle) w/ CSS for animation. @gvn you prefer the latter? If so, I will ask @taisdesouzalessa to create the SVG.

**Comment by gvn** [comment]
Ok, that makes more sense. I'd say keep this if we plan on adding the dot animations later, but if we're not then go to just a single SVG rotated w. CSS.

**Comment by mmmavis** [comment]
@gvn coooool. yea let's keep this then. Now it's code review time for you 👀

**Comment by mmmavis** [comment]
@gvn https://foundation-mofostaging-pr-1093.herokuapp.com/fellowships/directory/

File: .eslintrc.yaml [hunk 1]
```diff
@@ -13,3 +13,4 @@ parserOptions:
 
 rules:
   react/jsx-uses-vars: [2]
+  react/jsx-uses-react: [2]
```
Comment (mmmavis) [inline .eslintrc.yaml]: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-react.md

without this `source/js/components/loading-indicator/loading-indicator.jsx` will throw a linting error

**Comment by mmmavis** [comment]
☕️ 🍰

**Pull request merged**